Registry/Registry.EM_COMMON

```diff
@@ -231,7 +231,7 @@ i1       real   ph_tendf       ikj     dyn_em      1         Z
 i1       real   ph_save        ikj     dyn_em      1         Z 
 
 # Potential Temperature
-state    real   th_phy_m_t0    ikj      dyn_em      1         -     ihd "t"      "perturbation potential temperature theta-t0" "K"
+state    real   th_phy_m_t0    ikj      dyn_em      1         -     ih  "t"      "perturbation potential temperature theta-t0" "K"
```

Comment:
Why has this line been changed?

Reply:
@mgduda 
The addition of the 'd' is recent in the code (sometime after V4.0) and so was on the develop branch, from where I created this branch. This is an oversight by me. I'll add it back.